sql/core/src/test/scala/org/apache/spark/sql/sources/CreateTableAsSelectSuite.scala

```diff
@@ -170,23 +170,6 @@ class CreateTableAsSelectSuite extends DataSourceTest with SharedSparkSession {
     }
   }
 
-  test("disallows CREATE EXTERNAL TABLE ... USING ... AS query") {
```

Comment:
This is not possible as parser will deny `CREATE EXTERNAL TABLE ... USING`

Reply:
Is `parser will deny` different from `ParseException`? Can we change only two lines (185~186)?

Reply:
Maybe the error message was different. Let me revert back and fix it.

Reply:
+1